quicklogic/qlf_k4n8/tests/counter_16bit/counter_16bit.pcf

```diff
@@ -1,18 +1,18 @@
-set_io enable user_IN_T[0]
-set_io reset user_IN_T[1]
-set_io count[0] user_OUT_T[8]
-set_io count[1] user_OUT_T[9]
-set_io count[2] user_OUT_T[10]
-set_io count[3] user_OUT_T[11]
-set_io count[4] user_OUT_T[12]
-set_io count[5] user_OUT_T[13]
-set_io count[6] user_OUT_T[14]
-set_io count[7] user_OUT_T[15]
-set_io count[8] user_OUT_T[16]
-set_io count[9] user_OUT_T[17]
-set_io count[10] user_OUT_T[18]
-set_io count[11] user_OUT_T[19]
-set_io count[12] user_OUT_T[20]
-set_io count[13] user_OUT_T[21]
-set_io count[14] user_OUT_T[22]
-set_io count[15] user_OUT_T[23]
+set_io enable A2F_GPIO2_15
```

Comment:
Why are we changing the pin-mapping csv that is leading to updating these IO constraints